app/views/users/new.html.erb

```diff
@@ -1,3 +1,18 @@
 <% provide(:title, 'Sign up') %>
 <h1>Sign up</h1>
-<p>This will be a signup page for new users.</p>
\ No newline at end of file
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_with(model: @user, local: true) do |f| %>
```

Comment:
local này có ý nghĩa gì

Reply:
local: true: Khi người dùng nhấn nút gửi, trang sẽ **tải lại** và gửi yêu cầu đến server. Kết quả sẽ được phản hồi bởi server và trang sẽ được cập nhật với phản hồi đó.
local: false (hoặc không chỉ định local), Rails sẽ gửi biểu mẫu bằng AJAX, tức là dữ liệu của biểu mẫu sẽ được gửi đến máy chủ mà **không cần tải lại toàn bộ trang**.